bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala

```diff
@@ -97,7 +97,7 @@ object Overheads extends (BlueprintBase => Option[Overheads]) {
     case _: GnirsBlueprintImaging       => SimpleOverheads(0.10, 10, 0.193).some
     case _: GnirsBlueprintSpectroscopy  => SimpleOverheads(0.25, 15, 0.080).some
 
-    case _: NifsBlueprint               => SimpleOverheads(0.25, 11, 0.175).some
+    case _: NifsBlueprintBase           => SimpleOverheads(0.25, 11, 0.175).some
```

Comment:
Yes, that's right. There are two subclasses of this, namely `NifsBlueprint` and `NifsBlueprintAO`. Before, I was just matching on `NifsBlueprint` so `NifsBlueprintAo` was not returning anything in the overhead lookup. Now it will return these `SimpleOverheads` for both.